.github/workflows/android-build.yml

```
@@ -0,0 +1,77 @@
+        export QT_ROOT=$HOME/Qt
+        export WORKDIR=$PWD
+        
+        # Download and set up Qt (using the same script from Travis)
+        ./travisScripts/qt-5.8-android
+        cat qt-5.8.0-android-travis.env
+        ls $QT_ROOT
+        source qt-5.8.0-android-travis.env
+        
+        # Persist Qt, SDK and NDK exports for subsequent steps
+        cat qt-5.8.0-android-travis.env >> $GITHUB_ENV
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)